### ISSUE TITLE
How to change font size, font weight 

### ISSUE THREAD

**Issue by souravongit** [felixbuenemann/xlsxtream#15]
Hi,
Is there any option to set the font size, font weight like this?
I have used it, very simple to use but my generated xlsx fonts are larger and bold automatically. which needs to be changed.
Here is my code
```
      tmp_file = Tempfile.new(["template", ".xlsx"])
      # Creates a new workbook and closes it at the end of the block
      Xlsxtream::Workbook.open(tmp_file.path) do |xlsx|
        xlsx.write_worksheet('Sheet1', :auto_format => true, :use_shared_strings => true) do |sheet|
          sheet << accepted_columns
          JobTemplate.all.each do |job_template|
            sheet << job_template.attributes.except(rejected_columns.map {|str| str}.join(',')).values_at(*accepted_columns)
          end
        end
      end
```
Please let me know if it can be done.
Thanks

**Comment by felixbuenemann** [comment]
Currently the font is hardcoded to Calibri 12 pt here:

https://github.com/felixbuenemann/xlsxtream/blob/master/lib/xlsxtream/workbook.rb#L110

It would be relatively easy to pass in the font options to the Workbook constructor and then insert them into the stylesheet:

```ruby
Xlsxtream::Workbook.new(path, font: { name: 'Arial', size: '10' })
```

Would you like to try implementing that feature?

**Comment by felixbuenemann** [comment]
@souravongit @satyaki1 I have just released xlsxtream 1.2.0 on rubygems which supports font customization.

**Comment by souravongit** [comment]
Sorry for the late reaction. Thank you very much. this is really a very nice and simple gem to generate **xlsx** file.
